go/cache/lru_cache_test.go

```diff
@@ -80,17 +72,14 @@ func TestSetWithOldKeyUpdatesValue(t *testing.T) {
 	cache.Set(key, someValue)
 
 	v, ok := cache.Get(key)
-	if !ok || v != someValue {
-		t.Errorf("Cache has incorrect value: %v != %v", someValue, v)
-	}
+	assert.True(t, ok && v == someValue, "Cache has incorrect value: expected %v, got %v", someValue, v)
 }
 
 func TestGetNonExistent(t *testing.T) {
 	cache := NewLRUCache[*CacheValue](100)
 
-	if _, ok := cache.Get("notthere"); ok {
-		t.Error("Cache returned a notthere value after no inserts.")
-	}
+	_, ok := cache.Get("notthere")
+	assert.False(t, ok, "Cache returned a notthere value after no inserts.")
```

Comment:
Is it worth capturing and reporting the actual value we got from the map?
```suggestion
	val, ok := cache.Get("notthere")
	assert.False(t, ok, "Cache returned a notthere value after no inserts.", "val=%v", val)
```